CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
+
+If you are reading this, we thank you in advance for willing to contribute to the Watson project! You are awesome.
+
+Before starting to contribute, please read the following:
+
+* [How to contribute](https://tailordev.github.io/Watson/contributing/how-to/)
+* [How to start hacking](https://tailordev.github.io/Watson/contributing/hack/)
+* [Pull request guidelines](https://tailordev.github.io/Watson/contributing/pr-guidelines/)
+* [Contributor Code of Conduct](https://tailordev.github.io/Watson/contributing/coc/)
```

Comment:
I am sorry I might be missing the point, but after digging around, I don't see to what relative URLs should point to? Urls from github pages (`contributing/hack/`) are different from repository files relative paths (`docs/contributing/hack.md`).